### DISCUSSION TITLE
Tapjoy Offerwall through mediation

### DISCUSSION THREAD

**Discussion by ghost** [distriqt/ANE-IronSource#45] [Q&A]
Hey,

We would like to use the tapjoy offerwall feature.
Since there is no ANE for tapjoy and the native air SDK is deprecated, we asked tapjoy and ironsource about the possibility to use the tapjoy offerwall through iron source mediation.

They confirmed that this is possible. However we are not able to configure the offerwall as a mediation in the ironsource dashboard. (We will ask ironsource about it).
Our tapjoy contact forwarded us to this page: https://dev.tapjoy.com/en/support/Offerwall-With-Mediation

Here it says that it is possible to use the mediation APIs to access the tapjoy offerwall.
We were not able to use this within the ironsource ANE with the tapjoy mediation ANE.

Is there any way to access the tapjoy API within your ANE?

Thanks for your help!
Best regards,

**Comment by marchbold** [comment]
Hi,

I haven't attempted to do this currently and we have no immediate plans to implement it. 

Feel free to reach out to our custom development team if you need the addition: airnativeextensions@distriqt.com

Cheers
